README.md

```diff
@@ -24,18 +24,18 @@ After configuration, Harpoon can be run through [Docker](https://docs.docker.com
 docker-compose up --build
 ```
 
-Doing so will result in an endpoint at http://127.0.0.1:<web_server_port>, where `web_server_port` was the values specified during configuration. A Postgres database is also accessible through the corresponding port specified during configuration a well.
+Doing so will result in an endpoint at http://127.0.0.1:<web_server_port>, where `web_server_port` was the value specified during configuration. A Postgres database is also accessible through the corresponding port specified during configuration a well.
```

Comment:
I think we should display "http://127.0.0.1:<web_server_port>" in code formatting?